ministryofjustice/laa-review-criminal-legal-aid
# Crimre 89 use client gem

## Description of change
Integrates review with the [types gem](https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas), and the [datastore client api gem ](https://github.com/ministryofjustice/laa-criminal-applications-datastore-api-client)

## Link to relevant ticket
[CRIMRE-89](https://dsdmoj.atlassian.net/browse/CRIMRE-89)

## Notes for reviewer
- removes the old dev api code as well

## How to manually test the feature
- to test locally you will need to set up the data store and post some data to it. See the [datastore repo](https://github.com/ministryofjustice/laa-criminal-applications-datastore) to do this
- after this is set up test the app as if you are a user:
  - view page
  - click case
  - assign to self
  - un assign
  - etc...
